Server/RoomInstance.cpp

```diff
@@ -11,7 +11,7 @@
 namespace server_data
 {
     RoomInstance::RoomInstance(
-        Server *server, unsigned short id, std::string name, std::string address, unsigned short port)
+        Server *server, unsigned short id, std::string name, std::string address, unsigned short port, short *configs)
```

Comment:
👀

Reply:
configurations are used to add modifier on room creation. There are raw pointer because they are used by the network.